test/functional/treesitter/inspect_tree_spec.lua

```diff
@@ -120,22 +120,24 @@ describe('vim.treesitter.inspect_tree', function()
   end)
 
   it('updates source and tree buffer windows and closes them correctly', function()
+    local name = t.tmpname()
+    n.command('e ' .. name)
     insert([[
       print()
       ]])
+    n.command('set filetype=lua | w')
 
     -- setup two windows for the source buffer
     exec_lua(function()
       _G.source_win = vim.api.nvim_get_current_win()
-      vim.api.nvim_open_win(0, false, {
+      _G.source_win_copy = vim.api.nvim_open_win(0, false, {
```

Comment:
what is this a "copy" of?

Reply:
```suggestion
      _G.source_win = vim.api.nvim_open_win(0, false, {
```

Reply:
Of the window above `source_win`. Maybe `copy` isn't the best name, because it is simply another window displaying the source buffer.

Reply:
`source_win2` or something like that seems less confusing